recipes/validators/meta.yaml

```diff
@@ -61,21 +62,19 @@ about:
   # It is strongly encouraged to include a license file in the package,
   # (even if the license doesn't require it) using the license_file entry.
   # See https://conda.pydata.org/docs/building/meta-yaml.html#license-file
-  license_file: LICENSE.txt
-  summary: 'Simple, fast, extensible JSON encoder/decoder for Python'
+  license_file: LICENSE
```

Comment:
Please also add `license: MIT`.
https://github.com/kvesteri/validators/blob/0.14.0/setup.py#L59 says it would be BSD, but the actual license is MIT:
https://github.com/kvesteri/validators/blob/0.14.0/LICENSE#L1